lib/loader.js

```diff
@@ -6,6 +6,10 @@ const path = require('path');
 const linter = require('./linter.js');
 const resolver = require('oas-resolver');
 const yaml = require('js-yaml');
+const glob = require('glob');
+const util = require('util');
+
+const globAsync = util.promisify(glob);
```

Comment:
`util.promisify` is available since node 8 which is set as a minimum requirement in the README.md but I could avoid it when needed. It's just convenient here.